__tests__/fungible-token.spec.ts

```diff
@@ -0,0 +1,287 @@
+  });
+
+  test('Simple transfer', async () => {
+    await runner.run(async ({ft, ali, root}) => {
+      const initialAmount = new BN('10000');
+      const transferAmount = new BN('100');
+      await init_ft(ft, root, initialAmount);
+
+      // Register by prepaying for storage.
+      await registerUser(ft, ali);
+
+      await root.call(
+        ft,
+        'ft_transfer',
+        {
+          receiver_id: ali,
+          amount: transferAmount,
+        },
+        {attachedDeposit: '1'},
+      );
+
+      const rootBalance: string = await ft.view('ft_balance_of', {
+        account_id: root,
+      });
+      const aliBalance: string = await ft.view('ft_balance_of', {
+        account_id: ali,
+      });
+      expect(new BN(rootBalance)).toEqual(initialAmount.sub(transferAmount));
+      expect(new BN(aliBalance)).toEqual(transferAmount);
+    });
+  });
+
+  test('Can close empty balance account', async () => {
+    await runner.run(async ({ft, ali, root}) => {
+      await init_ft(ft, root);
+
+      await registerUser(ft, ali);
+
+      const result: boolean = await ali.call(
+        ft,
+        'storage_unregister',
+        {},
+        {attachedDeposit: '1'},
+      );
+
+      expect(result).toStrictEqual(true);
+    });
+  });
+
+  test('Can force close non-empty balance account', async () => {
+    await runner.run(async ({ft, root}) => {
+      await init_ft(ft, root, '100');
+      const unregister = async () =>
+        root.call(ft, 'storage_unregister', {}, {attachedDeposit: '1'});
+
+      await expect(unregister).rejects.toThrow();
+
+      const result = await root.call_raw(
+        ft,
+        'storage_unregister',
+        {force: true},
+        {attachedDeposit: '1'},
+      );
+
+      expect(result.receipts_outcome[0].outcome.logs[0]).toEqual(
+        `Closed @${root.accountId} with 100`,
+      );
+    });
+  });
+
+  test('Transfer call with burned amount', async () => {
+    await runner.run(async ({ft, defi, root}) => {
+      const initialAmount = new BN(10_000);
+      const transferAmount = new BN(100);
+      const burnAmount = new BN(10);
+      await init_ft(ft, root, initialAmount);
+      await init_defi(defi, ft);
+
+      await registerUser(ft, defi);
+
+      const result = await root
+        .createTransaction(ft)
+        .functionCall(
+          'ft_transfer_call',
+          {
+            receiver_id: defi,
+            amount: transferAmount,
+            msg: burnAmount,
+          },
+          {attachedDeposit: '1', gas: '150000000000000'},
+        )
+        .functionCall(
+          'storage_unregister',
+          {force: true},
+          {attachedDeposit: '1', gas: '150000000000000'},
+        )
+        .signAndSend();
+
+      expect(result.receipts_outcome[0].outcome.logs[1]).toEqual(
+        `Closed @${root.accountId} with ${
+          (initialAmount.sub(transferAmount)).toString()}`,
+      );
+
+      // Help: would be nice to have an API to avoid doing this
```

Comment:
This and cumbersome `result` parsing like `result.receipts_outcome[0].outcome.logs[1]` will be simplified by https://github.com/near/runner-js/issues/28